content/docs/communities/moderation/_index.md

```diff
@@ -95,21 +93,280 @@ in the message bar at the bottom:
+
+```
+!mjolnir list create spam my-community-spam-ban-list
+```
+
+You only need to create a ban list once, and can then add as many users and
+servers as you want to that ban list. You can also create as many ban lists as
+you want.
+
+Communities often create two ban lists: one for spam, and one for code of
+conduct infringements.
+
+### Subscribing to ban lists
+
+Ban lists are a clever mechanism that allows moderation teams to ban users for
+different motives (e.g. one list for `spam` and one for `coc`). Such a
+distinction can be useful when several communities want to collaborate together.
+
+Not all communities will share a similar Code of Conduct, but a lot of them will
+agree on what is spam. Being able to subscribe to another community's spam list
+means your own community will be protected from spammers the other community has
+already met, all while observing different code of conducts.
+
+To subscribe to a public ban list, you need to retrieve the address of this
+list. That list being technically nothing more than a Matrix room, its address
+follows the usual `#room_name:server.tld` format.
+
+Then to make mjolnir follow this list, you need to issue the following command
+in its control room
+
+```
+!mjolnir watch <room alias/ID>
+```
+
+For example to subscribe to the `#matrix-org-coc-bl:matrix.org` ban list
+maintained by the Matrix Foundation, you would issue the following command
+
+```
+!mjolnir watch #matrix-org-coc-bl:matrix.org
+```
+
+### Publishing ban lists
+
+From a technical perspective, a ban list is just a regular Matrix room filled
+with moderation hidden messages, technically called events. As such, you can
+give it a public address, and make it publicly accessible like you would for any
+other room.
+
+### Shadow-banning
 
-## Removing users from your community
+Shadow-banning is a controversial technique that should be used a last resort to
+deceive persistent spammers. It must not be used as part of routine moderation,
+and the Matrix Foundation strongly discourages using it.
 
-- Permanently or not
-- Redacting messages en masse
-- Banning users, banning servers
-- A word on shadow banning
+Shadow-banned users will try to send messages or various events to a homeserver,
+which will return successful responses. The messages and various events will not
+be propagated in the rooms. This corresponds to allowing the persistent spammer
+to shout in the void, without realising it.
 
-## Working together
+Shadow-banning is not part of the Matrix specification itself, which is why it
+is implemented in the non-standard [Admin API of Synapse](https://matrix-org.github.io/synapse/latest/usage/administration/admin_api/index.html).
 
-- Subscribing to ban lists
-- Publishing yours
+Homeserver administrators who want to resort to shadow banning can read about it
+[in Synape's documentation](https://matrix-org.github.io/synapse/latest/admin_api/user_admin_api.html#controlling-whether-a-user-is-shadow-banned).
```

Comment:
I'd maybe omit this section entirely?

Reply:
Fixed in [4d55d99](https://github.com/matrix-org/matrix.org/pull/1617/commits/4d55d999d3bff150df01f55a68ddb3ce10afaaa9)